core/domain/summary_services.py

```diff
@@ -150,8 +179,15 @@ def get_learner_collection_dict_by_id(
 
 
 def get_displayable_collection_summary_dicts_matching_ids(collection_ids):
-    """Returns a list with all collection summary objects that can be
-    displayed on the library page as collection summary tiles.
+    """Returns a list of dict with all collection summary objects from
+    collection ids.
+
+    Args:
+        collection_ids: list(str). A list of collection id
+
+    Return:
+        list(dict). A list of dict of collection summary objects that can be
```

Comment:
list(dict). Each element in this list is a collection summary dict. These elements are returned in the same order as that given in collection_ids.